packages/synapse-react-client/jest.config.cjs

```diff
@@ -19,17 +19,17 @@ module.exports = {
       moduleNameMapper: {
         '\\.(css|less|scss)$': 'identity-obj-proxy',
         // Mock SVGs, loaded by SVGR https://react-svgr.com/docs/jest/
-        '\\.svg$': '<rootDir>/test/testutils/svg.js',
+        '\\.svg$': '<rootDir>/src/testutils/MockSvg.js',
       },
       transformIgnorePatterns: [
         `node_modules/(?!(?:.pnpm/)?(${esModules.join('|')}))`,
       ],
-      setupFilesAfterEnv: ['<rootDir>/test/setupTests.ts'],
+      setupFilesAfterEnv: ['<rootDir>/src/testutils/setupTests.ts'],
       resetMocks: false,
     },
   ],
   reporters: [
-    '<rootDir>/test/testutils/LogForFailedTestsOnlyReporter.ts',
+    '<rootDir>/src/testutils/LogForFailedTestsOnlyReporter.cjs',
```

Comment:
This was always a `cjs` file (it had no type annotations). Jest has not been configured to transform this file, so the extension was updated to denote that it is in fact a CommonJS file.